tests/pact/keysets.repl

```diff
@@ -203,3 +203,54 @@
  "Scoped acquire of O succeeds"
  "Capability acquired"
  (test-capability (O)))
+
+;;
+;; keyset formats
+;;
+
+(env-exec-config ["EnforceKeyFormats"])
+(env-data
+ { 'bad: ['foo]
+ , 'short: ["12440d374865bdf0a3349634a70d1317fc279e7e13db98f2199ac5e7378975"]
+ , 'long: ["12440d374865bdf0a3349634a70d1317fc279e7e13db98f2199ac5e7378975eaea"]
```

Comment:
Ditto for shortening this long one by one character.  Since we're operating in the string domain it's characters, not after-decoded bytes, that are the fundamental unit.

Reply:
Manually checked.